routes/api/v1/playlists.js

```diff
@@ -119,13 +119,34 @@ router.get('/:playlistId/favorites', (req, res) => {
 router.delete('/:playlistId/favorites/:favoriteId', (request, response) => {
   let playlistId = request.params.playlistId
   let favoriteId = request.params.favoriteId
-  database('playlists_favorites').where({playlist_id: playlistId, favorite_id: favoriteId}).del()
-  .then(favorite => {
-    if (favorite > 0) {
-      response.status(204).send();
-    }
+
+  database('playlists_favorites').where({playlist_id: playlistId})
+  .then(playlists => {
+    if (playlists.length === 0) {
```

Comment:
If you have any feedback on how I can make these conditionals/validation checks better, I am happy to make the changes.

Reply:
I think it looks great!